examples/tutorials/o_transition_states.py

```diff
@@ -0,0 +1,26 @@
+import autode as ade
+
+# Use ORCA DFT optimisations
+ade.Config.lcode = 'xtb'
+ade.Config.hcode = 'orca'
+
+if not (ade.methods.ORCA().available and ade.methods.XTB().available):
+    exit('This example requires an ORCA and XTB install')
+
+# Locating transition states (TSs) in autodE requires defining a reaction.
+# For example, the TS for a key step in a Beckmann rearrangement can be
+# calculated with
+r1 = ade.Reactant('_data/Beckmann/reactant.xyz', charge=1)
+p1 = ade.Product('_data/Beckmann/product.xyz', charge=1)
+p2 = ade.Product('_data/Beckmann/water.xyz')
```

Comment:
Assume charge=0 is default?

Reply:
Yep, explained in `f_molecule_io.py` so I think this should be fine, as long as people go in order